### PR TITLE
sharing saturday info

### PR DESCRIPTION
**Pull request by makorzen**

### CODE REVIEW

**Comment by bast** [comment]
Shouldn't it be Sunday?

File: makorzen.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Party at Samuel's this Saturday!!
```
Comment (bast) [inline makorzen.txt:1]:
```suggestion
Party at Samuel's this Sunday!!
```

Comment (SamuelKo1607) [inline makorzen.txt:1]: good to know

Comment (SamuelKo1607) [reply makorzen.txt:1]: EVERYDAY!!!!!

**Pull request merged**